apps/website/src/components/Carousel.tsx

```diff
@@ -71,6 +71,7 @@ export default function Carousel({ title, sizes, type, ...props }: CarouselProps
             <HStack w="full" overflow="hidden">
                 <HStack
                     w="full"
+                    overflowX="auto"  // Enable horizontal scrolling
```

Comment:
It looks like the arrows buttons don't work properly after scrolling. I'm testing it with my laptop. Could you try it?

Reply:
You are right -- I forgot to test the arrows buttons and they were not working.

I've added a new commit that should fix this :) (at least on my laptop in several browsers, both the arrows and the scrolling should be working now)

Reply:
Hey @intldds, they are working now. But when I scroll and then use the arrows the scrollbar changes its size and cannot be used to reach all the items anymore. Can you see it?